utils/config.env

```diff
@@ -5,13 +5,13 @@
 # over time to construct other
 # values required by this repository
 #####################################################
-CONFLUENT=6.1.0-0
-CONFLUENT_DOCKER_TAG=6.1.x-latest
-CONFLUENT_SHORT=6.1
+CONFLUENT=6.2.0-0
```

Comment:
@awalther28 I think this change might have been mistakenly brought in by the retargeting (maybe the branch needs a rebase?) IIUC this file was not touched by this PR, so this needs to be reverted.  This applies to a few files:

- utils/config.env
- security/README.md
- replicator-schema-translation/docs/conf.py
- microservices-orders/docs/conf.py
- README.md
- ccloud/docs/config/*
- a few others...